pkg/project/app_classic/local.env

```diff
@@ -10,4 +10,8 @@ AEM_PUBLISH_PASSWORD=admin
 AEM_PUBLISH_HTTP_URL=http://127.0.0.1:4503
 AEM_PUBLISH_DEBUG_ADDR=0.0.0.0:14503
 
-AEM_DISPATCHER_HTTP_URL=http://127.0.0.1
+AEM_DISPATCHER_IP=127.0.0.1
+AEM_DISPATCHER_HTTP_URL=http://${AEM_DISPATCHER_IP}
+
+AEM_DOMAINS=host.docker.internal author.aem.local
```

Comment:
why `host.docker.internal` here? it's rather managed itself by Docker

Reply:
You have right, just my mistake